src/styles/card.scss

```diff
@@ -0,0 +1,114 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  font-family: $font;
+}
+
+body {
+  margin: 0;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 100%;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+
+  &__img {
```

Comment:
The `&__img` element should not have a fixed width and height unless it is a requirement for all images to strictly be this size. If the images are of varying dimensions, consider using more flexible properties like `max-width` and `max-height` to ensure the images are responsive and maintain their aspect ratio.